Makefile

```diff
@@ -46,7 +47,7 @@ md-to-godoc-gen:
 
 .PHONY: clean
 clean:
-	rm -rf cover.out cover.html lint.log fmt.log
+	rm -rf cover.out cover.html lint.log fmt.log jaeger-ui-build
```

Comment:
This doesn't really belong to this commit. I hope it's OK to leave this change here.